pkg/ebpf/events/events.go

```diff
@@ -172,29 +171,38 @@ func capturePolicyEvents(ringbufferdata <-chan []byte, log logr.Logger, enableCl
 					continue
 				}
 
+				srcIP := utils.ConvByteToIPv6(rb.SourceIP).String()
+				srcN, srcNS := utils.GetPodMetadata(srcIP)
```

Comment:
nit: use `srcName` instead, same for below `destName`

Reply:
I think better to handle the error in `GetPodMetadata`, if it returns error, we should add a warning, and continue to log without name and namespace.
Maybe encapsulate all the code changes in a separate function like 
```
func logPolicyEventWithK8sMetadata(srcIP, destIP, logger) {
  1. get the metadata for srcIP, if error, add a log.Info("failed to get name and namespace for srcIP xxx')
  2. get the metadata for destIP, if error, add a log.Warning("failed to get name and namespace for destIP xxx")
  3. utils.LogFlowInfo(...)
}
```

Reply:
Right now, my logic is that if the dest name and namespace is "" then I assume it is an external service destination so should I still handle the error of if there is no name and namespace for dest IP?